src/credentials/tests/TestFabricTable.cpp

```diff
@@ -79,6 +79,11 @@ class ScopedFabricTable
     chip::Credentials::PersistentStorageOpCertStore mOpCertStore;
 };
 
+// These two fabric ids are supposed to be the same; only the trust root is
+// different.
```

Comment:
why two constants though? why not a shared fabric id name? Otherwise reading through code if I see `x = kNode1Constant, y = kNode2Constant` I naturally assume they are different because different names ... and then I could go back up to see "equal on purpose".

Why not name this `constexpr FabricId kSharedFabricId` or something like that?

Reply:
These tests do not care that they are the same, it's not relevant. We just need a way to read the fabric id for a test NOC.

Reply:
Basically we just need a constant for the fabric id for test NOC AA_BB, instead of directly pasting this number all over the tests.

And I think it's preferable for tests that aren't concerned with the root of trust to also not have to concern themselves with whether or not the root of the trust is "shared" between test NOCs, or not.